### ISSUE TITLE
Taxonomy trouble-shooting

### ISSUE THREAD

**Issue by davidwolinsky** [giles-v/ghost-theme-dont-die#31]
There are ~40 interviews left on the "bad" spreadsheet [https://docs.google.com/spreadsheets/d/1sM1vaywl04wk3o15NL48VHtvvK7A0_kLsVOAPXm8lOw/edit#gid=1593171367] - interviews that don't have completed profiles b/c for various reasons I'm not satisfied with where to file them. The actual number to be sorted is lower because there are some who I don't have an age for, and just need to sit down and guesstimate or reach out to someone and ask them.

The bigger thing to broach is the potential to rethink the taxonomy in a macro or micro sense - whether DD is now in a place where it can present this data in a different way for sorting. My goal has always been for this to be a resource, and for people "outside" of games to be able to poke through and explore and learn. But I realize the majority audience will be hardcore enthusiasts, and the occasional curious people from other disciplines (IE, museum curators and restaurant-labor activists are two literal examples). 

My hesitation is creating categories that will only house one or two interviews, even though overtime obviously those can be filled as I do more interviews that would fall in that bucket. For example, "activists" and "technologists" both have only one. Lawyers only has two. 

A requested tag is simply "writers" or "critics," and that is a category I initially avoided in Don't Die just to make sure I was speaking to someone in a different aisle. But that is something I am starting to wade into more - and, for example, I wonder if "media theorists" and "media figures" should be condensed into one, bigger, more specifically named thing. 

There is another category I probably just need to make, which is for teenagers and kids. I guess I avoided making this category only because there *are* interviews with 12-year-olds. 

There's also an inconsistency in some of the names: "People Who..." vs. "Academics," "Lawyers," etc. "Entertainment Figures" is awkward. 

Input is appreciated and welcome.

**Comment by giles-v** [comment]
@davidwolinsky thanks for the detail.

I think this is pretty fungible and we can move stuff around, especially since we don't expose categories like this to SEO (yet). We could create some kind of "category groups" situation, but I think it would make more sense to just make the data model map directly onto what you want, I see no reason for layers of abstraction.

All of which is to say: can you make these changes yourself?

**Comment by davidwolinsky** [comment]
Absolutely. I believe this stemmed from whatever conversation was happening in Slack at the time - and I think here I was hoping to get some feedback / suggestions on things I'm not quite sure how to label. Are we waiting on me to make further changes to start making bio fields public both on interview pages and the homepage? That may have been what led to this thread.

But maybe these are questions best directed to Stanford and their archival staff? At a future date we will start having a pipeline for metadata, other things that will help massively with SEO. I want them to help pull weight on transcribing. 

I will mull - but if you have thoughts about any of the cases here please let me know. In the meantime, I will probably tinker on making these more consistent - though, for example, I'm not sure how to lump "media theorists" and "media figures" into one bigger thing. Maybe just "writers" and call it a day?
